packages/devtools_app/lib/src/standalone_ui/vs_code/devices.dart

```diff
@@ -38,7 +38,7 @@ class Devices extends StatelessWidget {
             children: [
               for (final device in devices)
                 _createDeviceRow(
-                  colorScheme,
+                  Theme.of(context),
```

Comment:
store theme in a var at the top of the build method so we don't have to call Theme.of(context) more than once

Reply:
Done!